documentation/airport-guides/airport-guide-directory.md

```diff
@@ -24,6 +24,7 @@ In order to help you learn the details of each particular airspace, an "airport
 - [KSEA - Seattle–Tacoma International Airport](ksea.md)
 - [KSTL - St. Louis Lambert International Airport](kstl.md)
 - [KTUS - Tucson International Airport](ktus.md)
+- [LKPR - Václav Havel Airport Prague](lkpr.md)
```

Comment:
I'd recommend using the airport name in `airportLoadList.js`, 'Vaclav Havel International Airport'

Reply:
Wilco